packages/auto-approve/src/valid-pr-schema.json

```diff
@@ -199,13 +199,13 @@
                     "enum": [
                       "/package\\.json$",
                       "/pom\\.xml$",
```

Comment:
`build.gradle`

Reply:
Also `*.sbt`.

Reply:
Before we add these, would you mind sending me some sample PRs so that I can confirm the rules/processes would look the same?

Reply:
Per our discussion, we'll add these later on.